wp-sw-cache/wp-sw-cache-main.php

```diff
@@ -47,6 +41,14 @@ public static function build_sw() {
       }
     }
 
+    // Seed cache with blank.gif if plugin is disabled or no files selected
+    // Because SW requires something to be cached to install and activate properly
```

Comment:
This is weird. Why don't you simply send an empty array. It will cause the contents of the cache to be evicted.